lib/nvme/nvme_internal.h

```diff
@@ -69,6 +69,12 @@
  */
 #define NVME_INTEL_QUIRK_WRITE_LATENCY 0x2
 
+/*
+ * The controller needs a delay before starts checking the device
+ * readiness, which is done by reading the NVME_CSTS_RDY bit.
```

Comment:
Reading of any PCI register will cause the problem (trigger PCIe completion timeout, which is a fatal error).